Add tip on how to install additional Python libraries (#838)
memgraph/docs: docs/reference-guide/query-modules/implement-custom-query-modules/overview.md

Custom modules developed via Memgraph Lab are located at
`/var/lib/memgraph/internal_modules`.

:::info

> If you need an additional Python library that is not already installed with
> Memgraph, check out our [guide on how to install
> it](/memgraph/how-to-guides/query-modules#how-to-install-external-python-libraries).

Reviewer: ```suggestion
If you need an additional Python library not included with
Memgraph, check out [the guide on how to install
it](/memgraph/how-to-guides/query-modules#how-to-install-external-python-libraries).

```
